app/Http/Controllers/SchoolsController.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Http\Controllers;
+
+use App\Helpers\VoivodeshipsHelper;
+use App\Http\Requests\SchoolRequest;
+use App\Http\Resources\SchoolResource;
+use App\Jobs\FetchSchoolsJob;
+use App\Models\School;
+use Illuminate\Bus\Batch;
+use Illuminate\Http\JsonResponse;
+use Illuminate\Http\RedirectResponse;
+use Illuminate\Support\Facades\Bus;
+use Illuminate\Support\Facades\Cache;
+use Inertia\Inertia;
+use Inertia\Response;
+use Symfony\Component\HttpFoundation\Response as Status;
+use Throwable;
+
+class SchoolsController extends Controller
+{
+    public function index(): Response
+    {
+        return Inertia::render("SchoolsPanel", ["schools" => SchoolResource::collection(School::all())]);
+    }
+
+    public function store(SchoolRequest $request): RedirectResponse
+    {
+        School::query()->create($request->validated());
+
+        return redirect()
+            ->back()
+            ->with("success", "School added successfully");
```

Comment:
Is the administration panel in Polish? If so, translate these messages

Reply:
done